src/db/OeClient.ts

```diff
@@ -59,6 +59,16 @@ class OEClient {
   private runProc(): Promise<any> {
     return new Promise((resolve) => {
       if (process.platform === "linux") {
+        const accessScript = `chmod +x ${path.join(
+          Constants.context.extensionPath,
+          "resources",
+          "oe",
+          "scripts",
+          "oe.sh"
+        )}`;
+        console.log(accessScript);
```

Comment:
do we need this log or is it just left after testing?

Reply:
I left it on porose. We log a few things here in this class, and for now I think it's ok. Latter when I will introduce my new implementation for Client logic, well fix all logging around here.